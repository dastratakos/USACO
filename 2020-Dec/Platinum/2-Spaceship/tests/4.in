60 4 60
101101000101110111001110100111111111110100110011010100110110
100110001111011001010110011111111111011111111011001011010101
101010011110101101010111111111111111111111111110011111111100
011111111011111101001110001111111101001101111110011011111100
001101011101110010101101111111111111001110111011111110110101
111111011011011100111010011100101001111111111110010111100111
111111001111111111101100100011001101000110111011000101011100
111011110011011100100010101011111100000110100111100101100011
100010101101001101011000111111011001010111110001111000011011
010101011110111100101000101110101110111110111011110011111011
100011011111111010011111001101101011100111011001001111011010
100101001000011101011111111111011111111111111100000011111011
110110011111101110101110000111110011101010111111101001111111
101101010111111011110010110010101010111111111111111110111100
011010011011100010011111001101101111010101011001001111001111
110110111111011111011111010111101011010011010111110110101111
101110111100111110011110001001000011111001011100111110101111
111110011111000101010110010001111111011111001011101110111110
001001111011111100011001101000111110110111111011000101001111
101100011111111111111011110000111101101110111111110110010111
101111010111110110011001111011111111111110111111110011111100
011011100010111101110110110011011101011001011111111101101011
111101101111101111011100111111000111111111110011011011100101
111111100111011100100010110001111101010101110101111101100101
101100011111111101010110010111100111101011011010100011011110
101101011011011110000101011111111101100111100011110010111011
111110011110001111010111010001011110110111110101111010011101
101010111100110110111101010011111010101010111100111011100111
111001111011101111011110101011011111011010000011000101110111
111111110111101001010111010111101111111110011111101111111111
100111011110101011111111100111111101111011101010100010011110
110001100111011110110011111110101101001111101111101111111110
100010010101101111010011011101010101011101110111101011001011
011010111011110111100111101000010110010101111101111110101110
000111110111101011101011110111011111110010101011111100101011
000001111101111111111011011010101010111010100011101111111011
010101110011001001010100101110101011111011011011111110111011
111001011011100110111010100100100111001111010001111001101111
110101111001111000101111001111111101011001010111000111110010
101011110100111111011001111101111111011101100001111110101111
010110101111111111010010111110111111100011011000010010110010
011101111101100110111111101110001111101111010010111101101111
011101010010111111110101100110110111101100001111011001010011
111011001111111101111111011111110010111110110011111100110011
110100100001100100000010100011110101111001100101111110011111
101110000101111000111111100101111111111110110011101110110001
111101011111011110101111101101011110111110011111011110110011
111111001011001011111001110111010111111101111110111111011100
011111111101110111110101101011011011011101101101011111010100
110111010110011011010111110101001111011111011011111111011111
111101011011101100001010011001110101100101010010110110100110
111110000111100110111001011111111011110010111110000011011101
111110010111110111111101011111101101000001001011111010111111
111111111001111111111111101101001111101011101101011110110110
111111101001110101011011111011110111101001110111011111111111
111011001110111100111111001111101100011111110111100100111011
110101011100110111000000000000111101100111101111011111111011
000000101010101000011110010011111010001011011101101011111101
001111111101100010111011100101111111111111110001011111110111
100110101101001001111011100110111111100011111111001011111100
3 32 3 1
3 32 1 44
3 32 3 11
3 32 1 36
3 32 4 32
3 32 4 36
3 32 3 60
3 32 3 3
3 32 1 43
3 32 3 10
3 32 3 41
3 32 1 23
3 32 4 35
3 32 4 55
3 32 1 42
3 32 4 15
3 32 3 59
3 32 2 21
3 32 1 59
3 32 3 54
3 32 2 58
3 32 4 9
3 32 2 15
3 32 4 55
3 32 2 37
3 32 3 49
3 32 2 56
3 32 2 43
3 32 2 8
3 32 4 19
3 32 4 12
3 32 1 49
3 32 2 51
3 32 2 24
3 32 2 50
3 32 4 43
3 32 4 59
3 32 4 41
3 32 2 23
3 32 4 49
3 32 1 17
3 32 4 22
3 32 4 12
3 32 1 34
3 32 1 1
3 32 3 32
3 32 2 22
3 32 1 59
3 32 4 40
3 32 1 37
3 32 3 32
3 32 2 41
3 32 1 14
3 32 1 49
3 32 3 2
3 32 4 33
3 32 3 50
3 32 2 48
3 32 4 41
3 32 3 32
