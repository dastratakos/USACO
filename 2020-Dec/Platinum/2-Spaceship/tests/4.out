228818915
198807528
739150434
613754310
48494
57813
247277017
66785949
533911910
658665057
772132471
482303781
50956
52565
468527929
52925
307346566
287404304
680162765
67378045
764487128
51026
414733445
52565
640535115
510761842
515702458
217788997
27436267
37687
55777
312587125
366800893
412945547
848536378
48317
54142
51439
609138325
44879
246978011
43065
55777
474585306
523115735
566685329
561446722
680162765
48143
845001938
566685329
419034417
795753826
312587125
234103946
56172
812941472
637652388
51439
566685329
